src/views/organisationType.njk

```diff
@@ -4,14 +4,16 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
+{% set title = 'Organisation Type' %}
+
 {% block backLink %}
   {{ govukBackLink({
   text: "Back",
   href: "/"
 }) }}
 {% endblock %}
 {% block content %}
-  <form method="post" action="/validate-organisation-type" id="organisationForm">
+  <form method="post" action="" id="organisationForm">
```

Comment:
why did we remove the action value for this form but not the others ?

Reply:
Ohh yes, good catch! I've removed as it gets populated later down